pkgs/development/python-modules/mahotas/default.nix

```diff
@@ -0,0 +1,32 @@
+{ buildPythonPackage, fetchFromGitHub, nose, pillow, scipy, numpy, imread, stdenv }:
+
+buildPythonPackage rec {
+  pname = "mahotas";
+  version = "1.4.2";
+
+  src = fetchFromGitHub {
+    owner = "luispedro";
+    repo = "mahotas";
+    rev = "v${version}";
+    sha256 = "1d2hciag5sxw00qj7qz7lbna477ifzmpgl0cv3xqzjkhkn5m4d7r";
+  };
+
+  patches = [ ./disable-impure-tests.patch ];
+
+  propagatedBuildInputs = [ numpy imread pillow scipy ];
+  checkInputs = [ nose ];
+
+  checkPhase= ''
+    python setup.py test
+  '';
+
+  disabled = stdenv.isi686; # Failing tests
```

Comment:
Are those the same tests that you disabled?

Reply:
I copied this from the original expression. I just built the expression from the `pkgsi686Linux` attr set and the package itself seems to build, but the tests fail, so instead we might want to build this against `i686`, but disable the check phase, right?

Reply:
Well if there are additional "real" test failures, its probably best to leave it as it is now. I just thought it may be worth checking if those tests that failed where the same you disabled for 64 bit.